Formula/kyuubi.rb

```diff
@@ -0,0 +1,53 @@
+class Kyuubi < Formula
+  desc "Serverless SQL on data warehouses and lakehouses"
+  homepage "https://kyuubi.apache.org/"
+  url "https://www.apache.org/dyn/closer.lua?path=kyuubi/kyuubi-1.6.1-incubating/apache-kyuubi-1.6.1-incubating-bin.tgz"
+  mirror "https://archive.apache.org/dist/kyuubi/kyuubi-1.6.1-incubating/apache-kyuubi-1.6.1-incubating-bin.tgz"
+  version "1.6.1-incubating"
+  sha256 "a64a6e5d71b39b0e494597282493250df1c74714fc2bba5057946926c0a32ba2"
+  license "Apache-2.0"
+  head "https://github.com/apache/kyuubi.git", branch: "master"
+
+  depends_on "apache-spark" => :test
```

Comment:
You're hardcoding the path in the install block. It's not test-only

Reply:
Does it work if I move spark home to the test part? I verified locally and it said `Error: kyuubi is missing test dependencies: apache-spark`. How can I make it automatically install test dependencies?

Reply:
seems OK now

Reply:
yeah, that is way to install test dependency, you did correctly.

Reply:
Thank you for the confirmation